product/add_remove_fulfillment.js

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
+  // [START retail_add_remove_fulfillment_places]
+
+  // Imports the Google Cloud client library.
+  const { ProductServiceClient } = require('@google-cloud/retail').v2;
+  const utils = require('./setup_cleanup');
+
+  process.env['GOOGLE_APPLICATION_CREDENTIALS'] = './sa.json';
```

Comment:
please remove this line GOOGLE_APPLICATION_CREDENTIALS should not be overwritten here